src/sentry/static/sentry/app/views/organizationGroupDetails/groupMerged/mergedItem.tsx

```diff
@@ -53,23 +56,21 @@ const MergedItem = createReactClass({
         return;
       }
 
-      this.setState({
-        [key]: stateForId[key],
-      });
+      this.setState(prevState => ({...prevState, [key]: stateForId[key]}));
     });
-  },
+  };
 
-  handleToggleEvents() {
+  handleToggleEvents = () => {
     const {fingerprint} = this.props;
     GroupingActions.toggleCollapseFingerprint(fingerprint);
-  },
+  };
 
   // Disable default behavior of toggling checkbox
-  handleLabelClick(e) {
-    e.preventDefault();
-  },
+  handleLabelClick(event: React.MouseEvent) {
```

Comment:
Does this also need to be a bound function (arrow func)?

Reply:
I don't think so, because it uses only the event